src/content/docs/validator/external-chains/hyperliquid.mdx

```diff
@@ -73,6 +73,8 @@ The visor binary spawns and manages the child node process.
 
 For faster bootstrapping, use a known reliable peer. Can be found [here](https://github.com/hyperliquid-dex/node?tab=readme-ov-file#mainnet-non-validator-seed-peers).
 
+**Note:** If you don't find peers for testnet, you can still start the node, but it may take some time to bootstrap.
```

Comment:
any approx time they should expect to wait that we can mention?

Reply:
not really sure for this. This was implicit based on this note in their docs "It may take a while as the node navigates the network to find an appropriate peer to stream from". If you think it's confusing though I can just remove this line